### DISCUSSION TITLE
Apps list - too expanded - would like a tighter few with smaller row sizes

### DISCUSSION THREAD

**Discussion by jddcef** [NexovaDev/UpdateHub#281] [Ideas]
Hi, reading through the list of apps, it's really painful, great app though.
But I would like a simple list perhaps a tabular form. Or at least more apps show per page, the spacing is too sparse, a tighter cosier layout would be better. Or at least some option to customize.

**Comment by onyx-nxt** [comment]
Will try to improve layout, I know that on wide monitors it looks a little wonky.
